src/sentry/templates/sentry/integrations/jira-server-config.html

```diff
@@ -27,9 +27,17 @@
 {% block title %} {% trans "Jira-Server Setup" %} | {{ block.super }} {% endblock %}
 
 {% block main %}
-{% if form %}
-<h3>{% trans "Step 2: Connect Sentry with your App" %}</h3>
+<h3>{% trans "Connect Sentry with your App" %}</h3>
   <p>{% trans "Add your Jira Server App credentials to Sentry." %}</p>
+  <p class="alert alert-block flex">
+    <i class="icon icon-exclamation"></i>
+    <span>
+        {% blocktrans %}
+        You must complete the <a href="https://docs.sentry.io/workflow/integrations/jira-server/">required steps</a>
```

Comment:
So this is more of a user experience comment than anything, but @markstory when I looked at the image of this, I wasn't clear that `required steps` was a link. Is that typical in our product for links to have that format? If so that's fine. Just pointing out, I didn't notice the link when I looked at the image.

Reply:
I'm going to have to talk to Sara or someone in design about that. I agree that the link is very not visible right now.